packages/@webex/plugin-meetings/src/locus-info/index.ts

```diff
@@ -110,6 +112,27 @@ export default class LocusInfo extends EventsScope {
     // return value ignored on purpose
     meeting.meetingRequest
       .getLocusDTO({url})
+      .catch((e) => {
```

Comment:
anyway, this fix doesn't do anything, because inside getLocusDTO() we are returning the error instead of throwing, so when request to Locus fails, getLocusDTO resolves with the error instead of rejecting. I'll add a fix for it.

Reply:
fixed